src/cli.ts

```diff
@@ -59,14 +58,20 @@ const cli = meow(
 )
 
 export const options = {
```

Comment:
This is great! It improves startup time, but I'm wondering how much it affects the overall execution time of the command? So we can understand the trade-offs of the solution 👀

Reply:
It shouldn't affect anything. It defers loading to when it's needed instead of pulling everything in memory from the get go. Code execution following the dynamic import proceeds like usual. It just gets to executing a lot faster since it doesn't have to load in all the other commands and their dependencies.

Only downside of dynamic importing AFAIK is the fact that tree shaking won't be available for the commands and their dependencies.